test/deps.test.ts

```diff
@@ -25,3 +21,15 @@ export {
     type Stub,
     stub,
 } from "https://deno.land/std@0.211.0/testing/mock.ts";
+
+export async function convertToUint8Array(
+    data: Iterable<Uint8Array> | AsyncIterable<Uint8Array>,
+) {
+    const values = [] as number[];
```

Comment:
smallest nit of all times
```suggestion
    const values: number[] = [];
```